### DISCUSSION TITLE
SAMM 15. to V2

### DISCUSSION THREAD

**Discussion by hward6** [owaspsamm/samm-suite#34] [Ideas]
Has there been a translation guide for someone to use their results from SAMM v1.5 and matrix it over into the V2 assessment? 

We have a lot of v1.5 assessments and will be asking our officers to fill out v2 assessments this year. It would be a great help if there was a matrix that existed for this to get them started on the transition. Has anyone else implemented this at their organizations?

**Comment by SebaDele** [comment]
hi,
No detailed mapping was created to map 1.5 to 2.0.
But an overview is available here: https://owaspsamm.org/mapping-versions/
This should help you?
Kind regards
Seba

**Reply by hward6** [reply]
Seba, Thank you for writing back. We have seen the logic mapping but it didn't really help in the sense of mapping the previous assessment responses into the new V2 format. I have listed requested as a feature, but will continue to work towards mapping it internally. Thank you for responding :)
